### PR TITLE
Update application FAQ

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Update application FAQ
Add the meta copy doc

## QA

- Open the copy doc in the meta copy docs
- Check the code change matches the copy change.

### CODE REVIEW

File: templates/careers/application/faq.html [hunk 1]
```diff
@@ -45,7 +47,7 @@ <h2 class="p-faq-stepped-list__title">Employment</h2>
...
+                <p>In general we expect our people to be able to travel 2-4 times a year to attend our Sprints. Our Sprints are in-person events where our functions come together to plan roadmaps, share learnings, collaborate and work on initiatives&hellip; and simply have great fun!  The locations for these sprints vary but are usually between EMEA and AMER and they usually last 1 week, occasionally 2 depending on your involvement. For some roles i.e. Sales, Field, Alliances etc there will be a more regular requirement to travel. Please make sure that you ask about the specific requirements during your interview with the team.  If you have any restrictions on travel please, please let us know.If you have any restrictions on travel please, please let us know.</p>
```
Comment (lorumic) [inline templates/careers/application/faq.html:50]: Do I see double?
```suggestion
                <p>In general we expect our people to be able to travel 2-4 times a year to attend our Sprints. Our Sprints are in-person events where our functions come together to plan roadmaps, share learnings, collaborate and work on initiatives&hellip; and simply have great fun!  The locations for these sprints vary but are usually between EMEA and AMER and they usually last 1 week, occasionally 2 depending on your involvement. For some roles i.e. Sales, Field, Alliances etc there will be a more regular requirement to travel. Please make sure that you ask about the specific requirements during your interview with the team.  If you have any restrictions on travel please, please let us know.</p>
```

**Comment by anthonydillon** [comment]
@lorumic Your right sorry, fixed

File: templates/careers/application/faq.html [hunk 2]
```diff
@@ -45,7 +47,7 @@ <h2 class="p-faq-stepped-list__title">Employment</h2>
...
+                <p>In general we expect our people to be able to travel 2-4 times a year to attend our Sprints. Our Sprints are in-person events where our functions come together to plan roadmaps, share learnings, collaborate and work on initiatives&hellip; and simply have great fun!  The locations for these sprints vary but are usually between EMEA and AMER and they usually last 1 week, occasionally 2 depending on your involvement. For some roles i.e. Sales, Field, Alliances etc there will be a more regular requirement to travel. Please make sure that you ask about the specific requirements during your interview with the team.  If you have any restrictions on travel please, please let us know.</p>
```
Comment (lorumic) [inline templates/careers/application/faq.html:50]: Sorry, I just noticed that there is also an extra space. Feel free to disregard this and proceed with merging though.
```suggestion
                <p>In general we expect our people to be able to travel 2-4 times a year to attend our Sprints. Our Sprints are in-person events where our functions come together to plan roadmaps, share learnings, collaborate and work on initiatives&hellip; and simply have great fun!  The locations for these sprints vary but are usually between EMEA and AMER and they usually last 1 week, occasionally 2 depending on your involvement. For some roles i.e. Sales, Field, Alliances etc there will be a more regular requirement to travel. Please make sure that you ask about the specific requirements during your interview with the team. If you have any restrictions on travel please, please let us know.</p>
```

**Review by lorumic — APPROVED** [review]
Noticed another tiny thing - sorry for not seeing it on first pass. Apart from that it's good to go.

**Pull request merged**